### PR TITLE
New libs

### PR DESCRIPTION
**Pull request by rubengees**
This PR covers

- [Android TextView Builder](https://github.com/klinker24/Android-TextView-LinkBuilder)
- [Anko](https://github.com/Kotlin/anko)
- [Kotterknife](https://github.com/JakeWharton/kotterknife)
- [Moshi](https://github.com/square/moshi)

which I use in one of my projects.

### CODE REVIEW

File: library/src/main/res/values/library_android_textview_linkbuilder_strings.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <string name="define_android_textview_linkbuilder"></string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_android_textview_linkbuilder_strings.xml]: please add `_int` to the define definition. 
without `_int` those libs will always be displayed. 


It should look like this: https://github.com/mikepenz/AboutLibraries/blob/develop/library/src/main/res/values/library_activeandroid_strings.xml#L4

Please also do this for the other 2 libs

Comment (rubengees) [inline library/src/main/res/values/library_android_textview_linkbuilder_strings.xml]: Ah! Forget about those, fixed ;)

**Comment by mikepenz** [comment]
thank you

**Comment by rubengees** [comment]
I added another lib which is use, but forget to initially include in this PR.

**Comment by mikepenz** [comment]
Oh ok thanks.

**Pull request merged**